package.json

```diff
@@ -80,7 +81,6 @@
     "react-portal": "^2.2.1"
   },
   "peerDependencies": {
-    "jquery": ">=2.1",
```

Comment:
please also update the readme to remove jquery from the example generated `npm install` command